src/pages/ProductDetail/components/ProductImage/ProductImage.js

```diff
@@ -0,0 +1,53 @@
+import React, { Component } from 'react';
+import './ProductImage.scss';
+
+class ProductImage extends Component {
+  constructor() {
+    super();
+    this.state = {
+      curImageIdx: 0,
+    };
+  }
+
+  /* idx의 맨 앞,끝에 왔을 때 어떤 방식으로  할지.. 함수를 두개? 아니면 조건을 하나에 다?*/
+  onClick = dir => {
```

Comment:
```suggestion
  onClick = direction => {
```
정확하게 적어주세요. dir은 directory의 약자처럼 보입니다